### DISCUSSION TITLE
[appDir] CSS Modules in Client Components

### DISCUSSION THREAD

**Discussion by gfortaine** [vercel/next.js#41782] [App Router]
Unfortunately, it looks like that CSS Modules don't work with Client Components (`"use client"` directive). I am preparing a PR with some failing tests that should have passed cc @leerob @shuding @huozhi

**Comment by oelbaga** [comment]
I had this issue today and it drove me bananas. I couldn't figure out what was happening. I was on 13.0.0 but when I updated to 13.0.1 canary it worked again! Made my day!

**Reply by icyJoseph** [reply]
Japp, can confirm that `next@13.0.1` has fixed this issue.

**Reply by gfortaine** [reply]
https://github.com/vercel/next.js/pull/42077
